libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -622,66 +632,38 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
     /**List item with radio button styles,check,uncheckupdates
      * event:any to avoid code duplication
      */
-    private _handleSingleSelect(event: any): void {
-        // clean up single selection items
-        if (event.target !== null &&
-            event.target !== undefined &&
-            this.selectionMode === 'single' &&
-            !this.selectRow) {
-            this.listItems.forEach((item) => {
-                if (item.radioButtonComponent !== undefined) {
-                    item.listItem.nativeElement.classList.remove('is-selected');
-                }
-            });
-            this._selectionModel.clear();
+    private _handleSingleSelect(event: Event, selectedItemId: string): void {
+        const singleSelect = event.target as Element;
+        const parent = singleSelect.closest('.fd-list__item');
```

Comment:
the same

Reply:
Hi Yuri, I have tried using the closest element from but it is not returning expected values.